### PR TITLE
Change log configuration path

### PR DESCRIPTION
**Pull request by LindsayHill**
Change to newer style log configuration path in PAM example

### CODE REVIEW

File: docs/source/authentication.rst [hunk 1]
```diff
@@ -76,7 +76,7 @@ repo. The following is a sample auth section in the config file for the PAM back
     use_ssl = True
     cert = /path/to/ssl/cert/file
```
Comment (nmaludy) [inline docs/source/authentication.rst]: Should we maybe put the default path to the cert and SSL key file in here also?

Comment (LindsayHill) [inline docs/source/authentication.rst]: Thought about it; decided not to. This SSL cert could be different to the default self-signed cert.

Comment (nmaludy) [inline docs/source/authentication.rst]: :+1:

**Comment by LindsayHill** [comment]
Related: https://github.com/StackStorm/st2-auth-backend-pam/pull/12

**Pull request merged**